### PR TITLE
Features/only display rental listing #162561695

### PR DESCRIPTION
**Pull request by callaghanc**
https://www.pivotaltracker.com/story/show/162561695

### CODE REVIEW

**Comment by callaghanc** [comment]
@software-project I've created a new PR for this story to correct the spelling of "features" in the branch name. Old PR: #1102.

**Comment by callaghanc** [comment]
I've reviewed this PR which is a recreation of @software-project's original PR, and I approve it in spirit. I can't add an official approving review since I created this PR! @akegan could you add an approving review to this on my behalf?

File: app/services/force/listing_service.rb [hunk 1]
```diff
@@ -211,5 +218,36 @@ def self.array_sort!(listing)
       listing['Units'] = stub_unit_data(listing['Units'])
       listing
     end
+
+    # TODO: Remove this method when we no longer need to stub data.
+    private_class_method def self.stub_rental_listing_data(listing)
```
Comment (akegan) [inline app/services/force/listing_service.rb]: hm, this tenure info should be available via the API now 🤔

Comment (callaghanc) [reply app/services/force/listing_service.rb]: if it's available then let's add it here

**Comment by software-project** [comment]
I removed stubbing for Tenure as it's not needed any more.

**Comment by callaghanc** [comment]
thanks @software-project, i'll re-QA the pivotal story for this

**Comment by software-project** [comment]
@callaghanc I just want to put some side notes regarding the tests. As we are using VCR, that has it's benefits and disadvantages I made tests a bit flexible. I didn't want to put exact numbers on expects, as they will change, when cassette is reloaded. Please check them and let me know if you fell like we need more scenarios covered.

**Comment by callaghanc** [comment]
@software-project thanks for the update! i'll take a look at the new tests and let you know

**Comment by callaghanc** [comment]
@software-project I've added the test updates we discussed. Can you review them, and if you approve, please add your approving review to this PR.

**Comment by callaghanc** [comment]
@akegan it looks like @software-project may be offline for the day. could you review this PR so we can get it merged? i added some work here so just my review is no longer sufficient to approve it

**Pull request merged**